vllm/model_executor/layers/fused_moe/fused_moe.py

```diff
@@ -949,8 +949,22 @@ def grouped_topk(
     num_expert_group: int = 0,
     topk_group: int = 0,
     scoring_func: str = "softmax",
-    e_score_correction_bias: Optional[torch.Tensor] = None
+    e_score_correction_bias: Optional[torch.Tensor] = None,
+    routed_scaling_factor: float = 1.0,
+    enable_fused: bool = True,
```

Comment:
When is enable_fused set? I don't see any changes other than the test. Do we need this?

Reply:
Yes it's for test.

Reply:
I have added `VLLM_TEST_DISABLE_FUSED_GROUPED_TOPK` env.

Since by default fused grouped_topk is enabled, `VLLM_TEST_DISABLE_FUSED_GROUPED_TOPK` defaults to 0.